pkg/globalmanager/runtime/worker.go

```diff
@@ -169,3 +183,46 @@ func createEnvVars(envMap map[string]string) []v1.EnvVar {
 	}
 	return envVars
 }
+
+// injectModelHotUpdateMount injects volume mounts when worker supports hot update of model
+func injectModelHotUpdateMount(pod *v1.Pod, object CommonInterface) {
+	hostPathType := v1.HostPathDirectoryOrCreate
+
+	var volumes []v1.Volume
+	var volumeMounts []v1.VolumeMount
+
+	modelHotUpdateHostDir, _ := filepath.Split(GetModelHotUpdateConfigFile(object, ModelHotUpdateHostPrefix))
+	volumeName := ConvertK8SValidName(ModelHotUpdateVolumeName)
+	volumes = append(volumes, v1.Volume{
+		Name: volumeName,
+		VolumeSource: v1.VolumeSource{
+			HostPath: &v1.HostPathVolumeSource{
+				Path: modelHotUpdateHostDir,
+				Type: &hostPathType,
+			},
+		},
+	})
+
+	volumeMounts = append(volumeMounts, v1.VolumeMount{
+		MountPath: ModelHotUpdateContainerPrefix,
+		Name:      volumeName,
+	})
+
+	injectVolume(pod, volumes, volumeMounts)
+}
+
+func GetModelHotUpdateConfigFile(object CommonInterface, prefix string) string {
+	return strings.ToLower(filepath.Join(prefix, object.GetNamespace(), object.GetObjectKind().GroupVersionKind().Kind,
+		object.GetName(), ModelHotUpdateConfigFile))
+}
+
+// setModelHotUpdateEnv sets envs of model hot update
+func setModelHotUpdateEnv(workerParam *WorkerParam) {
+	workerParam.Env["MODEL_HOT_UPDATE"] = "true"
+	s := workerParam.ModelHotUpdate.PollPeriodSeconds
+	if s == 0 {
```

Comment:
How to different the two cases below:
1. users specify `pollPeriodSeconds` with the value `0` 
2. users leave it blank.

Reply:
fix it

Reply:
`if s <= 0 || s > 1800 {
  s = 60
}`

Reply:
``` 
        s := workerParam.ModelHotUpdate.PollPeriodSeconds  
	if s < 0 || s > 1800 {  
		return fmt.Errorf("field pollPeriodSeconds must be greater than 0 and less than or equal to 1800")  
	} else if s == 0 {  
		klog.Warningf("checking that field pollPeriodSeconds is 0, the system will use the default value: 60")  
		s = 60  
	}  
	workerParam.Env["MODEL_POLL_PERIOD_SECONDS"] = strconv.Itoa(s)  
```